app/services/reporting/total_user_count_report.rb

```diff
@@ -45,6 +46,12 @@ def new_user_count
       end
     end
 
+    def total_fully_registered
+      Reports::BaseReport.transaction_with_timeout do
+        RegistrationLog.where('registered_at > ?', end_date).where.not(registered_at: nil).count
```

Comment:
whoops my bad I got this wrong in my previous suggestion, we need registered *before* this date not after
```suggestion
        RegistrationLog.where('registered_at <= ?', end_date).where.not(registered_at: nil).count
```